FluentAssertions.Net40/ObjectAssertionsExtensions.cs

```diff
@@ -47,8 +47,8 @@ public static class ObjectAssertionsExtensions
             {
                 object deserializedObject = CreateCloneUsingBinarySerializer(assertions.Subject);
 
-                EquivalencyAssertionOptions<T> defaultOptions = new EquivalencyAssertionOptions<T>()
-                    .IncludingAllRuntimeMembers();
+                EquivalencyAssertionOptions<T> defaultOptions = EquivalencyAssertionOptions<T>.Default()
```

Comment:
With the advent of global configuration you will need to decide if you want this respecting that configuration or not.  Changing it this way at the moment so that it at least matches XML serialization.